relay/src/setup.rs

```diff
@@ -9,6 +9,13 @@ pub fn check_config(config: &Config) -> Result<()> {
         );
     }
 
+    if config.relay_mode() == RelayMode::Proxy && config.processing_enabled() {
+        anyhow::bail!(
+            "Processing mode cannot be enabled while in the proxy mode. \
+            Please disable processing or switch to managed mode."
```

Comment:
This might incentivize users to enable processing in managed mode, so I would not even mention it here. Side note: Should `Processing` be a separate mode, i.e. variant of `RelayMode`, instead of a boolean flag?

```suggestion
            "
```